tests/policies/test_policies.py

```diff
@@ -161,7 +162,10 @@ def test_normalize_features_extractor(obs_space: gym.Space) -> None:
 
     for i in range(10):
         obs = th.as_tensor([obs_space.sample()])
-        obs = preprocessing.preprocess_obs(obs, obs_space)
+        # TODO(juan) the cast below is because preprocess_obs has too general a type.
```

Comment:
Agreed -- if you make the PR in SB happy to review it.

Reply:
Unfortunately there's no easy way to do this since python does not allow excluding types (so tensor is returned for any space except Dict), and the version of gym currently used in SB3 does not have support for generic spaces, so we cannot specify the types of elements in the observation space (i.e. what is returned by `sample()`).  Left this on https://github.com/DLR-RM/stable-baselines3/issues/1065